services/app-api/esbuild.config.ts

```diff
@@ -34,17 +35,19 @@ export default () => {
                             '.esbuild/.build/collector.yml'
                         )
                         let contents = fs.readFileSync(filePath, 'utf8')
+                        // Do we want a default value here or do we want to assert that NR_LICENSE_KEY always has a value
+                        // I don't think this solution is good, but this effectively mimics the existing behavior
                         contents = contents.replace(
                             '$NR_LICENSE_KEY',
-                            process.env.NR_LICENSE_KEY
+                            process.env.NR_LICENSE_KEY ?? 'undefined'
```

Comment:
@mojotalantikite Do we care if NR_LICENSE_KEY is not present in an environment? Is this something that we want to crash the app, by having an assert? 

Maybe we want to add a chosen key and maybe log an error?